utils.js

```diff
@@ -47,13 +49,47 @@ export function renderShip(starship) {
     li.appendChild(shipCategory);
 
     button.textContent = 'Add to Megacart';
-    
+
+    button.addEventListener('click', () => {
```

Comment:
wonderful work, Ryan! Also, don't forget to clean up some white spaces as well when you can so you have the cleanest code :)